app/models/text_field_type.rb

```diff
@@ -1,8 +1,11 @@
+require 'jsonb_accessor'
 class TextFieldType < FieldType
   attr_accessor :text
+  jsonb_accessor :data, text: :string
 
   validates :text, presence: true, if: :validate_presence?
   validate :text_length, if: :validate_length?
+  validate :text_unique, if: :validate_uniqueness?
```

Comment:
Presence just checks if the value is an empty string or not, uniqueness requires searching all `FieldItems` text.